test/gov/swagger_validation.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+CONTAINERID=/swagger-validator-v2
+VALIDATOR_URL=http://172.18.0.8:8080
```

Comment:
I dont think you can count on this IP address. On my dev box, I get variations of IP@s for containers on the e2edev network. I think you are going to need code like this: https://github.com/open-horizon/anax/blob/master/test/gov/run_kube.sh#L86 to get the validation container's IP address.